Compile error enabling SERIAL_STATS_MAX_RX_QUEUED/SERIAL_STATS_DROPPED_RX on AT90USB

When I compile Marlin 1.1.8 (release) with SDSUPPORT enabled in configuration.h
and SERIAL_STATS_MAX_RX_QUEUED / SERIAL_STATS_DROPPED_RX in configuration_adv.h I get the following errors:
```
In file included from sketch\Marlin.h:46:0,

                 from sketch\Marlin_main.cpp:247:

sketch\Marlin_main.cpp: In function 'void loop()':

Marlin_main.cpp:14572: error: 'customizedSerial' was not declared in this scope

             SERIAL_ECHOLNPAIR("Dropped bytes: ", customizedSerial.dropped());

                                                  ^

sketch\serial.h:52:76: note: in definition of macro 'SERIAL_PROTOCOLPAIR'

 #define SERIAL_PROTOCOLPAIR(name, value)    (serial_echopair_P(PSTR(name),(value)))

                                                                            ^

sketch\serial.h:61:40: note: in expansion of macro 'SERIAL_PROTOCOLLNPAIR'

 #define SERIAL_ECHOLNPAIR(pre,value)   SERIAL_PROTOCOLLNPAIR(pre, value)

                                        ^

sketch\Marlin_main.cpp:14572:13: note: in expansion of macro 'SERIAL_ECHOLNPAIR'

             SERIAL_ECHOLNPAIR("Dropped bytes: ", customizedSerial.dropped());

             ^

Marlin_main.cpp:14576: error: 'customizedSerial' was not declared in this scope

             SERIAL_ECHOLNPAIR("Max RX Queue Size: ", customizedSerial.rxMaxEnqueued());

                                                      ^

sketch\serial.h:52:76: note: in definition of macro 'SERIAL_PROTOCOLPAIR'

 #define SERIAL_PROTOCOLPAIR(name, value)    (serial_echopair_P(PSTR(name),(value)))

                                                                            ^

sketch\serial.h:61:40: note: in expansion of macro 'SERIAL_PROTOCOLLNPAIR'

 #define SERIAL_ECHOLNPAIR(pre,value)   SERIAL_PROTOCOLLNPAIR(pre, value)

                                        ^

sketch\Marlin_main.cpp:14576:13: note: in expansion of macro 'SERIAL_ECHOLNPAIR'

             SERIAL_ECHOLNPAIR("Max RX Queue Size: ", customizedSerial.rxMaxEnqueued());

             ^

exit status 1
'customizedSerial' was not declared in this scope
```
Compiles fine with the SERIAL_STATS_MAX_RX_QUEUED / SERIAL_STATS_DROPPED_RX defines commented
I'm guessing there's a couple of missing checks related to "customizedSerial" in Marlin_main.cpp ?

[config.zip](https://github.com/MarlinFirmware/Marlin/files/1602417/config.zip)

--- a/Marlin/Marlin_main.cpp
+++ b/Marlin/Marlin_main.cpp
@@ -14284,13 +14284,15 @@ void loop() {
           card.closefile();
           SERIAL_PROTOCOLLNPGM(MSG_FILE_SAVED);
 
-          #if ENABLED(SERIAL_STATS_DROPPED_RX)
-            SERIAL_ECHOLNPAIR("Dropped bytes: ", customizedSerial.dropped());
-          #endif
+          #ifndef USBCON
+            #if ENABLED(SERIAL_STATS_DROPPED_RX)
+              SERIAL_ECHOLNPAIR("Dropped bytes: ", customizedSerial.dropped());
+            #endif
 
-          #if ENABLED(SERIAL_STATS_MAX_RX_QUEUED)
-            SERIAL_ECHOLNPAIR("Max RX Queue Size: ", customizedSerial.rxMaxEnqueued());
-          #endif
+            #if ENABLED(SERIAL_STATS_MAX_RX_QUEUED)
+              SERIAL_ECHOLNPAIR("Max RX Queue Size: ", customizedSerial.rxMaxEnqueued());
+            #endif
+          #endif // !USBCON
 
           ok_to_send();
         }

--- a/Marlin/SanityCheck.h
+++ b/Marlin/SanityCheck.h
@@ -275,8 +275,8 @@
   #elif TX_BUFFER_SIZE && (TX_BUFFER_SIZE < 2 || TX_BUFFER_SIZE > 256 || !IS_POWER_OF_2(TX_BUFFER_SIZE))
     #error "TX_BUFFER_SIZE must be 0, a power of 2 greater than 1, and no greater than 256."
   #endif
-#elif ENABLED(SERIAL_XON_XOFF)
-  #error "SERIAL_XON_XOFF is not supported on USB-native AVR devices."
+#elif ENABLED(SERIAL_XON_XOFF) || ENABLED(SERIAL_STATS_MAX_RX_QUEUED) || ENABLED(SERIAL_STATS_DROPPED_RX)
+  #error "SERIAL_XON_XOFF and SERIAL_STATS_* features not supported on USB-native AVR devices."
 #endif
 
 /**

